docs/contributors/code/release.md

```diff
@@ -48,18 +48,21 @@ You should check that folks are able to install the new version from their Dashb
 
 Once released, all that's left to do is writing a release post on [make.wordpress.org/core](https://make.wordpress.org/core/). You can find some tips on that below.
 
-### Writing the Release Notes and Post
+### Documenting the release with the Release Notes and Post
 
-Documenting the release is comprised of the following steps:
-1. Curating the changelog
-2. Selecting the release highlights
-3. Drafting the release post
-4. Running the performance tests
-5. Publishing the post
+<div class="callout callout-info">
+Documenting the release **is a group effort between the release manager, Gutenberg core team developers, and designers** , comprised of a series of sequential steps that, because of the amount of people involved and the coordination required, need to adhere to a timeline between the RC and stable releases.
+</div>
+
+1. Curating the changelog - Wednesday after the RC release to Friday
+2. Selecting the release highlights - Friday to Monday
```

Comment:
I might add at this point that this is when design is brought in to then help with the assets. This should help make it clearer for design when they will be involved and what their focus should be.

Reply:
Thanks, added it to the corresponding section below.